apps/mobile/src/components/PostCard/PostBodyProposal/PostBodyProposal.js

```diff
@@ -135,7 +139,7 @@ export default function PostBodyProposal ({
         const optionVotes = proposalVotesArray.filter(vote => vote.optionId === option.id)
         const avatarUrls = optionVotes.map(vote => vote.user.avatarUrl)
         return (
-          <TouchableOpacity key={`${option.id}+${currentUserVotesOptionIds.includes(option.id)}`} style={[styles.proposalOption, votingComplete && styles.completed, proposalStatus === PROPOSAL_STATUS_DISCUSSION && styles.discussion, proposalStatus === PROPOSAL_STATUS_VOTING && styles.voting, proposalStatus === PROPOSAL_STATUS_CASUAL && styles.casual, currentUserVotesOptionIds.includes(option.id) && styles.selected, votingComplete && styles.completed, votingComplete && highestVotedOptions.includes(option.id) && styles.highestVote]} onPress={isVotingOpen(proposalStatus) && !votingComplete ? () => handleVote(option.id) : () => {}}>
+          <TouchableOpacity key={`${option.id}+${currentUserVotesOptionIds.includes(option.id)}`} style={[styles.proposalOption, votingComplete && styles.completed, proposalStatus === PROPOSAL_STATUS_DISCUSSION && styles.discussion, proposalStatus === PROPOSAL_STATUS_VOTING && styles.voting, proposalStatus === PROPOSAL_STATUS_CASUAL && styles.casual, currentUserVotesOptionIds.includes(option.id) && styles.selected, votingComplete && styles.completed, votingComplete && highestVotedOptions.includes(option.id) && styles.highestVote]} onPress={isVotingOpen(proposalStatus) && !isVoting && !votingComplete ? () => handleVote(option.id) : () => {}}>
```

Comment:
Please put this logic in a const (or a `useMemo` call if needed) above. I apologize ahead of time if this is still in progress, and can wait to review until it's marked ready if so. Just please do mark it to ready when it is if that is the case. 

_One more comment while I'm here though, and this was my fault for misdirecting you:

`addProposalVoteMutationFetching` should be `addProposalVoteFetching`, and the same for the other two. We drop the `*Mutation*` part once we're outside of graphql land and in component land if that makes sense._